inject/README.md

```diff
@@ -1,6 +1,20 @@
-# DInject
+# Avaje Inject
 APT based dependency injection for server side developers - https://dinject.io
 
+### Example module use
+
+```java
+module org.example {
+
+  requires javax.inject;
+  requires org.slf4j;
```

Comment:
Do consumers actually need to add SL4J?

Reply:
No, I got that wrong ... because in the inject module-info I needed to make these two dependencies `requires transitive`.  Once I do that the consumers no longer need to specify `requires org.slf4j` or  `requires javax.inject;`.